dallinger/recruiters.py

```diff
@@ -872,6 +903,38 @@ def external_submission_url(self):
         self.prolificservice.log_response(response)
         return response
 
+    def screen_out_allowed(self, participants: list[Participant]) -> bool:
+        """
+        Check if all participants in a list of participants can be screened out.
+
+        Returns True if all participants can be screened out, False otherwise.
+        """
+        study = self.prolificservice.get_study(self.current_study_id)
+
+        if not study.get("is_custom_screening", False):
+            raise ProlificRecruiterException(
+                f"Prolific study (ID {self.current_study_id}) doesn't allow screening-out of participants. "
+                "Set 'is_custom_screening' to 'True' in your experiment config.txt file (or alternatively in "
+                "~/.dallingerconfig) to enable screening-out."
+            )
+
+        # Minimum wage thresholds: https://researcher-help.prolific.com/en/article/2273bd
+        prolific_min_wage_per_hour = 6
+        min_required_reward = (
+            median_time_spent_in_hours(participants) * prolific_min_wage_per_hour
+        )
+
+        for participant in participants:
```

Comment:
The `screen_out_allowed` needs to take the `bonus_per_participant` value (or whatever it's called) as an argument. The test is then simply 

```py
min_required_reward <= bonus_per_participant
```

No need to iterate over participants.

Reply:
Initially, I thought we wanted to check each single participant if it can be screened-out.

Reply:
Yes, I see that. Unfortunately I think the Prolific API works a different way